ui/app/AppLayouts/Profile/views/WalletView.qml

```diff
@@ -58,19 +58,24 @@ SettingsContentBase {
     }
 
     dirty: manageTokensView.dirty
-    ignoreDirty: stackContainer.currentIndex === manageTokensViewIndex
+    ignoreDirty: stackContainer.currentIndex === manageTokensViewIndex && !manageTokensView.advancedTabVisible
+
     saveChangesButtonEnabled: dirty
     toast.type: SettingsDirtyToastMessage.Type.Info
-    toast.cancelButtonVisible: false
-    toast.saveForLaterButtonVisible: dirty
-    toast.saveChangesText: qsTr("Apply to my Wallet")
-    toast.changesDetectedText: qsTr("New custom sort order created")
+    toast.cancelButtonVisible: manageTokensView.advancedTabVisible
+    toast.saveForLaterButtonVisible: !manageTokensView.advancedTabVisible
+    toast.saveChangesText: manageTokensView.advancedTabVisible ? toast.defaultSaveChangesText : qsTr("Apply to my Wallet")
+    toast.changesDetectedText: manageTokensView.advancedTabVisible ? toast.defaultChangesDetectedText : qsTr("New custom sort order created")
 
     onSaveForLaterClicked: {
         manageTokensView.saveChanges()
     }
     onSaveChangesClicked: {
-        manageTokensView.saveChanges()
+        if (manageTokensView.advancedTabVisible) {
+            // Save changes only when tab is active
```

Comment:
Why? This will prevent saving (applying) changes in the Assets/Collectibles tab

Reply:
> @benjthayer do we want to discard changes when leaving the advanced tab? I'm wondering if we should use a consistent approach for all the flows in the application, bc if not, it may be strange for the user. For example, in the `Profile Showcase`, changes are never discarded while moving between tabs. If here the issue is that the popup buttons text is different depending on the tab, maybe we need to consider this design and use the same texts. WDYT?

Hey @noeliaSD , for the profile showcase all of the tabs are highly related. The user will likely want to change settings across multiple tabs before committing those changes - so to make things more efficient for the user, it's fine to persist the same save panel across the tabs and not discard changes when the user moves from one tab to the next without saving. 

It is slightly different for the Manage Tokens view as each tab is functionally different - every tab has a different and relatively unrelated purpose. For that reason, I think it is fine for changes to be dismissed as the user tabs away as @Cuteivist shows in the above video. 

The one thing we still need to design (and this is on my to do list) is whether we block the user from moving away from a tab or settings area while the save changes panel is displayed and if moving away means the loss of any changes to settings. Discord applies a wiggle motion to the save panel and blocks the user moving away - so we need to globally consider designing something like this.

Hope that answers your question? But yes, we definitely need to explore a global solution / pattern for when the user tries to move away with the save panel visible.